### ISSUE TITLE
Issues with Ionic framework (v1)

### ISSUE THREAD

**Issue by indrimuska** [indrimuska/angular-moment-picker#173]
This is to group all issues related to Ionic framework and warn if they are fixed:

- [ ] #171
- [ ] https://github.com/indrimuska/angular-moment-picker/issues/51#issuecomment-299415567

**Comment by dementhius** [comment]
Hi @indrimuska I post you a gif where you can see what happens.

![calendar_error](https://cloud.githubusercontent.com/assets/4408471/25792644/13b5da36-33c8-11e7-95ee-f804b8834d0a.gif)

These are bower components, it could be a conflict with another library.

  "devDependencies": {
    "ionic": "driftyco/ionic-bower#1.1.1",
    "plugin": "^3.1.0"
  },
  "dependencies": {
    "ion-autocomplete": "~0.3.0",
    "ion-gallery": "~0.1.6",
    "moment": "2.18.1",
    "jquery": "*",
    "angular-utils-pagination": "*",
    "angular-nvd3": "*",
    "angular-sc-select": "*",
    "moment-picker": "0.9.11"
  }

And these are npm ones:
  "dependencies": {
    "angular-chart.js": "^1.0.3",
    "chart.js": "^2.4.0",
    "elementtree": "^0.1.6",
    "lodash": "^3.10.1",
    "plist": "^1.1.0",
    "protractor": "*"
  },
  "devDependencies": {
    "bower": "^1.3.3",
    "shelljs": "^0.3.0",
    "gulp": "^3.5.6",
    "gulp-concat": "^2.2.0",
    "gulp-minify-css": "^0.3.0",
    "gulp-rename": "^1.2.0",
    "gulp-sass": "^1.3.3",
    "gulp-flatten": "*",
    "gulp-header": "*",
    "gulp-jasmine": "*",
    "gulp-stylus": "*",
    "gulp-util": "*",
    "gulp-uglify": "*"
  },

Thx

**Comment by dkrychylskyy** [comment]
Thanks, but I'm a little confused what to do )

That's what I did:

1. I created a new project Ionic `ionic start myapp --v1`,
2. added moment-picker - `bower install moment-picker --save`
3. added in app.js moment-picker - `angular.module('starter', ['ionic', 'starter.controllers', 'starter.services', 'moment-picker'])`

and again I got the same error
```

> Uncaught Error: [$injector:modulerr] Failed to instantiate module starter due to:
> Error: [$injector:modulerr] Failed to instantiate module moment-picker due to:
> Error: [$injector:nomod] Module 'moment-picker' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.

```
Help me understand please

**Comment by dementhius** [comment]
Hey,

If you are using Gulp or Grunt you must compile  your library CSS and JS for you Ionic App main CSS and JS files. If not, you must add the paths in index.html to your library files in bower or node_modules folder. Then App will detect then.

**Comment by indrimuska** [comment]
Exactly, the answer of @dementhius should fix the @dkrychylskyy's issue, so I'm going to close it.

**Comment by indrimuska** [comment]
@dementhius regarding your issue, I've just created [this plunker](http://plnkr.co/edit/udGFZQ?p=preview) and it seems to work properly. Can you take a look at it?

**Comment by saidbouig** [comment]
@dementhius could you be more specific please?  i'm getting the same error with my Karma Unit testing

**Comment by Sasicumaar** [comment]
![image](https://user-images.githubusercontent.com/25055829/41645137-b2925bfa-748d-11e8-9674-929cf1202c03.png)

This will solve this issue
